filter/src/main/java/com/yalantis/filter/animator/FiltersListItemAnimator.kt

```diff
@@ -191,25 +188,25 @@ class FiltersListItemAnimator : SimpleItemAnimator() {
 
     override fun animateMove(holder: RecyclerView.ViewHolder, fromX: Int, fromY: Int,
                              toX: Int, toY: Int): Boolean {
-        var fromX = fromX
-        var fromY = fromY
+        var fx = fromX
```

Comment:
inconsistent variable naming: Before there was `fromX` and `toX`. Now it is `fx` and `toX`. I suggest reverting back to the old naming style

Reply:
Android Studio:Name shadowed: fromX

Reply:
Ah, I overlooked the parameter names. However, that warning won't create issues in this code. You could attach `@Suppress("NAME_SHADOWING")` to those 2 variables.

Technically the old 194 - 195 code could have been named better as the `fromX` and `fromY` variables are modified in the code. Maybe something like `_fromX` and `_fromY`

Regardless, the names of `fx` and `fy` don't fit the variable naming standard and should be changed